libs/blocks/section-metadata/section-metadata.js

```diff
@@ -34,6 +36,37 @@ function handleTopHeight(section) {
   const headerHeight = document.querySelector('header').offsetHeight;
   section.style.top = `${headerHeight}px`;
 }
+
+export function promoIntersectObserve(el, startPoint, options = {}) {
+  let abovePromoStart = false;
+  const io = new IntersectionObserver((entries, observer) => {
+    entries.forEach(async (entry) => {
```

Comment:
You aren't doing any async operations, you can remove the `async` keyword

Reply:
And anyway, `forEach` and `async` don't work together, so this definitely doesn't achieve anything.

Reply:
@mokimo @overmyheadandbody I have made the requested changes.